src/MockHandler.php

```diff
@@ -48,12 +48,12 @@ public function append()
     {
         foreach (func_get_args() as $value) {
             if ($value instanceof ResultInterface
-                || $value instanceof AwsException
+                || $value instanceof Exception
                 || is_callable($value)
             ) {
                 $this->queue[] = $value;
             } else {
-                throw new \InvalidArgumentException('Expected an Aws\ResultInterface or Aws\Exception\AwsException.');
+                throw new \InvalidArgumentException('Expected an Aws\ResultInterface or Aws\Exception.');
```

Comment:
This should be the PHP core exception, not aws\exception (we don't have a class named exception).  It will also need the following use statement added at the top: 
use Exception;